### DISCUSSION TITLE
Port of MVVMCross for Godot game engine.

### DISCUSSION THREAD

**Discussion by Ilnazz** [MvvmCross/MvvmCross#4788] [General]
Hello, thank you for amazing MVVM library. I would like to create a port of your library for the Godot game engine. What do you think about this idea, is it possible and how difficult is it?

**Comment by Cheesebaron** [comment]
Is it possible, probably.

How difficult it is, depends on which features of MvvmCross you want to support in GoDot.

You'd probably want to start with implementing the NavigationService and then perhaps making bindings work to some UI elements.

So to support a new platform you would need GoDot specific implementations for:

- MvxSetup
- IMvxNavigationService

That would be the bare minimum I think.

I am not super familiar with the inner workings of GoDot, so you are a bit on your own here.

**Reply by Ilnazz** [reply]
Thanks a lot for the reply!
